README.md

```diff
@@ -18,10 +18,10 @@ jobs:
     runs-on: ubuntu-latest
     name: Generate DevCycle PR Insights
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: DevCycleHQ/feature-flag-pr-insights-action@v2.0.4
+      - uses: DevCycleHQ/feature-flag-pr-insights-action@main
```

Comment:
this should document the current version, not main

Reply:
is that to ensure users are only using a released version